internal/featuretests/v3/routeweight_test.go

```diff
@@ -659,14 +659,11 @@ func routeweightedcluster(clusters ...weightedcluster) *envoy_route_v3.Route_Rou
 
 func weightedclusters(clusters []weightedcluster) *envoy_route_v3.WeightedCluster {
```

Comment:
not directly related to your PR but this looks identical to `weightedClusters` in `internal/featuretests/v3/envoy.go`, would be nice to remove this one.

Reply:
done!